Sources/PointFree/Account/Account.swift

```diff
@@ -906,6 +907,64 @@ private func inviteRowView(_ invite: TeamInvite) -> Node {
     ])
 }
 
+private func addTeammateToSubscriptionRow(_ data: AccountData) -> [Node] {
```

Comment:
this is the module to allow adding a teammate:

<img width="799" alt="image" src="https://user-images.githubusercontent.com/135203/63227742-c46a7d80-c1b7-11e9-8878-0db8a1678c69.png">

it only shows if you have no invites remaining on your subscription.